### PR TITLE
release: 0.8.1

### PR DESCRIPTION
**Pull request by mvidalgarcia**

### CODE REVIEW

File: templates/page.html [hunk 1]
```diff
@@ -276,12 +276,12 @@ <h3>Developers</h3>
...
-            <h3>REANA 0.8.0</h3>
-            <h6>30 Nov 2021</h6>
-            <p>Introduces support for running Snakemake workflows.</p>
-            <p>Improves cluster performance when running multiple parallel workflows.</p>
-            <p>Enables administrators to configure CPU and Disk user quotas.</p>
-            <p class="release-news-link"><a href="https://github.com/reanahub/reana/releases/tag/0.8.0">See full release notes</a></p>
+            <h3>REANA 0.8.1</h3>
+            <h6>15 Feb 2022</h6>
+            <p>Introduces support for job timeout limits.</p>
+            <p>Improves Cluster Health status page on the web interface.</p>
```
Comment (tiborsimko) [inline templates/page.html]: What about:

- Introduces custom job timeout limits for user workflows.
- Allows searching workspace files by name and visualising HTML reports.
- Improves Cluster Health status web page indicating cluster availability.

Comment (tiborsimko) [reply templates/page.html]: WRT illustrative picture, the cluster health could do, but I think that the screenshot of the custom job timeouts (as I have added it to the blog post) or of workspace file name search filtering (as you had it in the blog post) might be more illustrative.

(The thinking being that these features will be more frequently used by the researchers than checking out the cluster health page.)

Comment (mvidalgarcia) [reply templates/page.html]: Updated.

Replaced release picture with workspace search one.

File: LICENSE [hunk 2]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (C) 2018, 2019 CERN.
+Copyright (C) 2018-2022 CERN.
```
Comment (tiborsimko) [inline LICENSE]: It would be good to do `2018, 2019, 2020, 2021, 2022` here as well.

(See e.g. `reana-commons`.)

**Review by tiborsimko — APPROVED** [review]

**Pull request merged**